src/SFA.DAS.EAS.Web/Views/Shared/_Layout.cshtml

```diff
@@ -2,6 +2,7 @@
 @using SFA.DAS.Authorization
 @using SFA.DAS.Authorization.Mvc
 @using SFA.DAS.EAS.Web
+@using SFA.DAS.EAS.Web.Extensions

```

Comment:
Is this needed/part of this story?

Reply:
Nope...but it annoyed me so I changed it.  It caused prod error log entries so it should probably its own story,